examples/aggregator/client/aggregator.cc

```diff
@@ -103,8 +103,10 @@ int main(int argc, char** argv) {
     values.push_back(value);
   }
 
-  // Submit data sample.
-  submit_sample(stub.get(), bucket, indices, values);
+  // Submit data sample as many times as it takes to reach the hardcoded threshold.
+  for (int i = 0; i < 5; i++) {
+    submit_sample(stub.get(), bucket, indices, values);
+  }
```

Comment:
@ipetr0v let me know what you think of this. I can remove it if you prefer, but there must be a mechanism somewhere to allow for testing the actual logic that we are interested in (even if it's just the `run_example` script calling this binary 5 times).

Reply:
I think it's better than it was before.